content/en/user-manual/animation/anim-state-graph-assets.md

```diff
@@ -0,0 +1,171 @@
+    </tr>
+</tbody></table>
+
+It is possible to create multiple transitions between two animation states, which have different values and conditions set. The priority of these transitions can be reordered in the transition inspector after selecting a transitions arrow in the graph. The priority order determines which transition will be used by the state graph if multiple transitions have their conditions met.
+
+### Parameters
+
+The parameters of an anim state graph are variables which are used to control the flow of animations during runtime. These variables can be accessed via scripts and set to new values at any time. They are then the way in which users can control the behaviour of an entity's animation during its lifecycle.
+
+New parameters can be added to a state graph via the parameters panel on the left inspector:
+
+![Parameters][3]
+
+Each parameter has three variables which can be set:
+
+<table class="table table-striped">
+    <tbody>
+        <tr>
+            <th>Variable</th>
+            <th>Description</th>
+        </tr>
+    <tr>
+        <td>Name</td>
+        <td>The name that this parameter should be called by. This is used to find and set the parameter via script. Names must be unique per state graph.</td>
+    </tr>
+    <tr>
+        <td>Type</td>
+        <td>The type of variable that the parameter contains. One of; Boolean, Float, Integer or Trigger. The Trigger type acts as a Boolean but with the special property that it’s value is set back to false after it has been used to successfully activate a transition.</td>
+    </tr>
+    <tr>
+        <td>Default Value</td>
+        <td>The value of the parameters variable when the state graph launches.</td>
+    </tr>
+</tbody></table>
+
+The way in which they control the state graph is through the use of transition conditions. Each transition in the graph can have a list of conditions which define when a transition is usable by the system. A state will not be able to pass to another state through a given transition unless all of it’s conditions are met.
+
+Each condition consists of a conditional statement which compares the current value of a parameters variable to the given value in the condition using the designated operator. For example the following condition:
+
+![Condition][4]
+
+Can be used in the transition between the Idle and Jump animation states to ensure that a character only jumps when the ‘Jump’ parameter has been set to true via a script.
+
+### Layers
+
+So far animstategraph assets have been discussed in the context of editing a single animation state graph. It may sometimes be necessary however to have the animations of a single model driven by multiple separate state graphs, each with their own defined behaviour. An example could be animating a main character's movement and locomotion on a single layer, which animating it’s facial expressions on a separate layer that’s driven by its own state graph and parameters.
```

Comment:
```suggestion
So far, animstategraph assets have been discussed in the context of editing a single animation state graph. It may sometimes be necessary however to have the animations of a single model driven by multiple separate state graphs, each with their own defined behavior. An example could be animating a main character's movement and locomotion on a single layer, while animating its facial expressions on a separate layer that’s driven by its own state graph and parameters.
```